app/scripts/directives/overview/mobileClientRow.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+(function () {
+  angular.module('openshiftConsole').component('mobileClientRow', {
+    controller: [
+      '$scope',
+      '$filter',
+      '$routeParams',
+      'ProjectsService',
+      'DataService',
+      'APIService',
+      'MobileClientsService',
+      'ListRowUtils',
+      'Navigate',
+      'AuthorizationService',
+      MobileAppRow,
+
+    ],
+    controllerAs: 'row',
+    bindings: {
+      apiObject: '<',
+      state: '<',
+      bindings: '<'
+    },
+    templateUrl: 'views/overview/_mobile-client-row.html'
+  });
+
+  function MobileAppRow($scope, $filter, $routeParams, ProjectsService, DataService, APIService, MobileClientsService, ListRowUtils, Navigate, AuthorizationService) {
+    var row = this;
+    $scope.installType = '';
```

Comment:
Think we should pick one of `$scope` or `controllerAs` in a component and stick to it.

Reply:
sure